scripts/download_all_miz.py

```diff
@@ -0,0 +1,90 @@
+""" 
+Downloads all MIZ shapefiles from this link: https://usicecenter.gov/Products/ArchiveSearchMulti?table=DailyArcticShapefiles
+
+To run:
+pixi run python scripts/download_all_miz.py
+"""
+import os
+from datetime import datetime
+
+import requests
+from bs4 import BeautifulSoup
```

Comment:
We need to have our pixie.toml file updated too to include this dependency. Right?

Reply:
you're right, it's in the lock file since it's a dependency of something else, but it should be explicitly a proj dependency
`      - conda: https://conda.anaconda.org/conda-forge/noarch/beautifulsoup4-4.12.3-pyha770c72_0.conda`

nice catch!!!